include/prereq-build.mk

```diff
@@ -53,6 +53,18 @@ $(eval $(call TestHostCommand,ncurses, \
 	Please install ncurses. (Missing libncurses.so or ncurses.h), \
 	echo 'int main(int argc, char **argv) { initscr(); return 0; }' | \
 		gcc -include ncurses.h -x c -o $(TMP_DIR)/a.out - -lncurses))
+
+$(eval $(call TestHostCommand,libelf-dev, \
```

Comment:
I've been building OpenWrt for some time now and never needed libelf.{so,h}, what am I missing?

Reply:
It's been a prerequisite for a long time ((https://openwrt.org/docs/guide-developer/build-system/install-buildsystem), but without an explicit check I suspect it's missing for some people. Previously, it supports building kernels with STACK_VALIDATION enabled (for at least x86/64 target), but without it the kernel build doesn't fail, just prints a warning that most folks don't notice:
```
warning: Cannot use CONFIG_STACK_VALIDATION=y, please install libelf-dev, libelf-devel or elfutils-libelf-devel
```
I added this check unconditionally because we now will have two users for `libelf-dev`.

Reply:
I just checked on the latest buildbot container image and it's missing there as well.

```bash
buildbot@c2789e94c2f6:/$ apt list --installed | grep libelf
libelf1/stable,now 0.176-1.1 amd64 [installed,automatic]
```

The `libelf-dev` package is not installed by default

Try locally via:
```
user@dawn:~/src/openwrt/21.02$ sudo docker run -it --rm registry.gitlab.com/openwrt/buildbot/buildworker apt list --installed | grep libelf
libelf1/stable,now 0.176-1.1 amd64 [installed,automatic]
```

Reply:
Please create a PR here to add that package: https://gitlab.com/openwrt/buildbot

Ping @ynezz

Reply:
First gitlab PR, no idea if this is right: https://gitlab.com/openwrt/buildbot/-/merge_requests/1

Reply:
It's not correct, nobody is going to review patches on 3 different platforms. GitLab is currently only used for evaluation purposes. Ideally this should go over mailing list, thanks.

Reply:
> Please create a PR here to add that package: https://gitlab.com/openwrt/buildbot

I think, that openwrt.org/submitting-patches is good enough.

Reply:
Sorry for the extra loop. With added CI to buildbot i figured it's best to create it there directly

Reply:
Mirroring is setup from git.openwrt.org to gitlab.com

Reply:
> It's not correct, nobody is going to review patches on 3 different platforms. GitLab is currently only used for evaluation purposes. Ideally this should go over mailing list, thanks.

I can assure you nobody wants to submit patches on 3 different platforms either, as I've had to do in just the last week.

><aparcar[m]> guidosarducci: you can use your github account to login, if it's to much of a fuzz I'll do it

@aparcar Since you offered on IRC to take care of this, I'll ask if you can do that now. Thanks, it's been too much of a fuzz.

Reply:
@aparcar Thank you for updating the buildbots. I'll close this off assuming there are no other concerns.